autoload/lsp/buffer.vim

```diff
@@ -99,7 +99,7 @@ export def BufLspServerGet(bnr: number, feature: string = null_string): dict<any
 
   # LSP server is configured to be a provider for "feature"
   for lspserver in possibleLSPs
-    var has_feature: bool = lspserver.features->get(feature, false)
+    var has_feature: bool = lspserver.featureEnabled(feature)
```

Comment:
this is Not same meaning statement, it was going to check feature option.

Reply:
I think I understand now.

This first loop checks if any features have been _explicitly_ enabled by the user. 

The second loop checks if any default features are enabled?

I will revert this